modules/machine-user-infra-machines-ibm-z-kvm.adoc

```diff
@@ -121,6 +121,7 @@ The `os-variant` is case sensitive.
 <2> For `--location`, specify the location of the kernel/initrd on the HTTP or HTTPS server.
 <3> For `coreos.inst.ignition_url=`, specify the `worker.ign` Ignition file for the machine role. Only HTTP and HTTPS protocols are supported.
 <4> For `coreos.live.rootfs_url=`, specify the matching rootfs artifact for the `kernel` and `initramfs` you are booting. Only HTTP and HTTPS protocols are supported.
+<5> Optional: For `hostname`, specify the fully qualified host name of the client machine.
```

Comment:
@SNiemann15 Should we make this hostname? 

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#h